tar/bsdtar.c

```diff
@@ -1236,10 +1237,20 @@ configfile(struct bsdtar *bsdtar, const char *fname)
 	bsdtar->option_no_config_include_set =
 	    bsdtar->option_no_config_include;
 
-	/* If the file doesn't exist, do nothing. */
 	if (stat(fname, &sb)) {
-		if (errno == ENOENT)
-			return;
+		/* Missing file. */
+		if (errno == ENOENT) {
+			if (fromcmdline) {
+				bsdtar_errc(bsdtar, 1, errno,
+				    "Missing --configfile %s", fname);
```

Comment:
This prints `tarsnap: Missing -configfile foo: No such file or directory`, I think?  Maybe changing the errmsg string here to "Cannot read configuration file: %s" would be better?